pkg/agent/testdata/AKSUbuntu1604+K8S117/line33.sh

```diff
@@ -75,20 +75,20 @@ else
 fi
 
 installContainerRuntime
-
 installNetworkPlugin
 
+
```

Comment:
nit: any chance you can tweak the go templates to make this diff go away

Reply:
Which templates would that that be? This was just spit out from `make generate` so I assume it should be kept this way otherwise next person to run that will also get weird diffs.